e2e/common/common.go

```diff
@@ -107,7 +183,20 @@ func NewConfig(
 
 	var nodes []*node.Node
 	for i := 0; i < validators; i++ {
-		n := node.NewNode(int64(i), conf.BaseDir, conf.LoomPath, conf.ContractDir, genesisTmpl, yamlFile)
+		n := node.NewNode(int64(i), conf.BaseDir, loompathAbs, conf.ContractDir, genesisTmpl, yamlFile)
+		n.LogLevel = *LogLevel
+		n.LogDestination = *LogDest
+		n.LogAppDb = *LogAppDb
+		nodes = append(nodes, n)
+	}
+
+	loompathAbs2, err := filepath.Abs(loompath2)
+	if err != nil {
+		return nil, err
+	}
+
+	for i := validators; i < validators+altValidators; i++ {
+		n := node.NewNode(int64(i), conf.BaseDir, loompathAbs2, conf.ContractDir, genesisTmpl, yamlFile)
```

Comment:
We should print out what nodes are running `loom` and what nodes are running `loom2`